specs/api.md

```diff
@@ -190,6 +190,10 @@ Serwer zapisuje dane do tabeli `Encounters` wraz z odpowiednim `user_id`.
 
 `date` : `datatime`
 
+`processed_by_gis` : `bool`
```

Comment:
Czy możemy to zmienić na `processed_by_health_authority`? GIS to polski skrót i w połączeniu z angielskim robi się mało czytelne

Reply:
👍